pkgs/development/tools/database/pgcli/default.nix

```diff
@@ -18,7 +18,7 @@
 , mock
 }:
 
-buildPythonApplication rec {
+buildPythonPackage rec {
```

Comment:
Can you add a comment why `buildPythonPackage` is used here, so that no one is changing it back?